modules/ROOT/pages/anypoint-mq/2.x/anypoint-mq-listener.adoc

```diff
@@ -230,6 +230,31 @@ In many cases there's a single common service between the processing of messages
 
 For this scenario, both subscribers stop polling for messages as soon as the error count reaches the `errorsThreshold="10"` value, counting both `FTP:RETRY_EXHAUSTED` and `HTTP:SERVICE_UNAVAILABLE` errors. When the `tripTimeout` elapses, one of the subscribers polls for a message and uses it to test the circuit, enabling the polling for both subscribers if the processing of that message succeeds.
 
+
+== FIFO Queues
+
+FIFO queues are most suitable for single-consumer scenarios. When one consumer is accessing a message, all other consumers are blocked until the first batch is processed. No messages are delivered until all in-flight messages are acknowledged or not acknowledged. With message groups, multiple consumers can access messages in a FIFO queue. In this case, a single consumer accesses messages in a group and message order is preserved within the message group.
+
+----
+IMPORTANT: 
```

Comment:
You probably know by now that I have an unwarranted dislike of notice formats, so...your decision (as always)....

Is there a way to work this into the earlier text of the paragraph above? Something like, FIFO queues are most suitable...but you must use a single-thread flow configuration to preserve message processing order"

Reply:
It is really important, which is why I like to see it early on. Some studies suggest that some readers actually *ignore* notices, especially if they have read work that uses a ton of them for way too many things....